ansible/roles/nodejs/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Download NVM
+  shell: curl https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh --output nvminstall.sh
+  args:
+    creates: nvminstall.sh
+  become: yes
+  become_user: vagrant
+
+- name: Install NVM
+  shell: sh nvminstall.sh
```

Comment:
sh nvminstall.sh != ./nvminstall.sh

I've seen you do this before. `sh` is not needed to run shell scripts. I'm not sure if this will solve your problem or not in this case, but you almost never want to do this.